lib/rbkit.rb

```diff
@@ -106,15 +31,15 @@ def self.start_server(pub_port: DEFAULT_PUB_PORT, request_port: DEFAULT_REQ_PORT
   # profiling is not feasible.
   def self.start_profiling(pub_port: DEFAULT_PUB_PORT, request_port: DEFAULT_REQ_PORT,
```

Comment:
Previously, you could do things like `Rbkit.status` or `Rbkit.send_objectspace_dump` . These methods don't really make sense because there's no indication that a server has been started to send the status or object space dump. I think it makes more sense to put these methods on an instance of Rbkit::Server, so it would be: 

```ruby
server = Rbkit.start_profiling # or Rbkit.start_server
server.status
server.send_objectspace_dump
server.stop # or Rbkit.stop_server
```

I don't expect users to use these APIs since they'll be using the GUI to send messages that trigger these methods, still it provides a way to do it from code. We are using this a lot inside our tests.